wsd/ClientRequestDispatcher.cpp

```diff
@@ -1716,7 +1721,7 @@ bool ClientRequestDispatcher::handleMediaRequest(const Poco::Net::HTTPRequest& r
         LOG_TRC_S("Move media request " << tag << " to docbroker thread");
 
         std::string range = request.get("Range", "none");
-        docBroker->handleMediaRequest(std::move(range), socket, tag);
+        docBroker->handleMediaRequest(std::move(range), socket, tag, (bVTT ? "srt" : "url"));
```

Comment:
```suggestion
        docBroker->handleMediaRequest(std::move(range), socket, tag, (bVTT ? MEDIA_SRT : MEDIA_MP4));
```
Or something along this way should be better.